Jenkinsfile

```diff
@@ -0,0 +1,74 @@
+def dockerImage = "maven:3-jdk-8"
+def dockerArgs(env) { return "-e JAVA_TOOL_OPTIONS=-Duser.home=${env.WORKSPACE}"; }
+
+pipeline {
+    agent any
+
+    triggers {
+        cron('H H(20-23) * * 1-5')
```

Comment:
I think primer is quite "stable" in terms of code changes so we might drop cron trigger as well.

Reply:
ok